index.js

```diff
@@ -175,17 +177,17 @@ module.exports = function (config) {
    */
 
   // serve all the provider json
-  app.get('/providers', function (req, res, next) {
-    res.json(app.services)
+  koop.get('/providers', function (req, res, next) {
+    res.json(koop.services)
   })
 
   // serve up a provider
-  app.get('/providers/:provider', function (req, res, next) {
-    res.json(app.services[req.params.provider])
+  koop.get('/providers/:provider', function (req, res, next) {
```

Comment:
may not part of this PR, but can we move some of these routes to a separate module or controller? There's really just for Koop-UI as far as I can tell